### PR TITLE
macrocreator: Add version 5.3.2

### PR DESCRIPTION
**Pull request by snowman**

### CODE REVIEW

File: bucket/macrocreator.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+    "version": "5.3.2",
+    "description": "Free Automation Tool and Script Generator based on AutoHotkey language.",
+    "homepage": "https://www.macrocreator.com",
+    "license": {
```
Comment (Ash258) [inline bucket/macrocreator.json:5]: Do not use URL for valid SPDX identifiers

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/macrocreator.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    "hash": "03691b301a2d55742b5bbf1c6145a2963c0287428d58535c9b1b3596397db4d9",
+    "architecture": {
+        "64bit": {
+            "extract_dir": "MacroCreatorPortable\\x64\\MacroCreator"
+        },
+        "32bit": {
+            "extract_dir": "MacroCreatorPortable\\x86\\MacroCreator"
+        }
+    },
+    "shortcuts": [
```
Comment (Ash258) [inline bucket/macrocreator.json:16]: https://github.com/Pulover/PuloversMacroCreator/blob/master/MacroCreator.ahk#L152-L154

```suggestion
    "pre_install": "Copy-Item \"$persist_dir\\*\" \"$dir\" -Include 'UserEmailAccounts.ini', 'UserGlobalVars.ini' -ErrorAction SilentlyContinue",
    "uninstaller": {
        "script": "Copy-Item \"$dir\\*\" \"$persist_dir\" -Include 'UserEmailAccounts.ini', 'UserGlobalVars.ini' -ErrorAction SilentlyContinue -Force"
    },
    "shortcuts": [
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

**Comment by issaclin32** [comment]
I am merging this without update because
(1) The branch has been deleted
(2) Autoupdate and installation still works fine

We can just wait for automatic updates, and everything will be OK.

**Comment by rashil2000** [comment]
The branch still exists - https://github.com/follow-the-vine-to-get-to-the-melon/scoop-extras/tree/MacroCreator

**Comment by issaclin32** [comment]
> The branch still exists - https://github.com/follow-the-vine-to-get-to-the-melon/scoop-extras/tree/MacroCreator

Oops. I was wrong. I thought it was deleted when I found that I cannot edit the file.

**Comment by rashil2000** [comment]
Check back in 2 hours, Excavator will have run by then.

**Comment by snowman** [comment]
Thanks , I delete the `MacroCreator` branch now, in case you want to take over

**Comment by issaclin32** [comment]
> I would help out if you want, but now I restored deleted `MacroCreator` branch

You can delete the branch now, because it is no longer needed when the PR is merged.

**Comment by rashil2000** [comment]
image

**Comment by issaclin32** [comment]
> Excavator failed here
> 
> image

Thanks for the follow up.
Still not sure why `v$version` turns into `v1.1.1` 😕 
```
"autoupdate": {
        "url": "https://github.com/Pulover/PuloversMacroCreator/releases/download/v$version/PuloversMacroCreator-Portable.zip"
    }
```

**Comment by rashil2000** [comment]
Locally it seems to running fine:

image

**Comment by issaclin32** [comment]
Looks like there is a bug in the *Excavator*, which turns all new versions into `1.1.1`.

```
acrocreator: 5.4.1 (scoop version is 5.3.2) autoupdate available
Autoupdating macrocreator
Downloading PuloversMacroCreator-Portable.zip to compute hashes!
The remote server returned an error: (404) Not Found.
URL https://github.com/Pulover/PuloversMacroCreator/releases/download/v1.1.1/PuloversMacroCreator-Portable.zip is not valid
ERROR Could not update macrocreator, hash for PuloversMacroCreator-Portable.zip failed!
mu-editor: 1.1.0 (scoop version is 1.0.3)
kid3: 3.9.0 (scoop version is 3.8.7) autoupdate available
Autoupdating kid3
Could not find hash in https://sourceforge.net/projects/kid3/files/kid3/1.1.1
Downloading kid3-1.1.1-win32.zip to compute hashes!
The remote server returned an error: (404) Not Found.
URL https://downloads.sourceforge.net/project/kid3/kid3/1.1.1/kid3-1.1.1-win32.zip is not valid
ERROR Could not update kid3, hash for kid3-1.1.1-win32.zip failed!
languagetool-java: 5.6 (scoop version is 5.5) autoupdate available
Autoupdating languagetool-java
Downloading LanguageTool-1.1.1.zip to compute hashes!
The remote server returned an error: (404) Not Found.
URL https://languagetool.org/download/LanguageTool-1.1.1.zip is not valid
ERROR Could not update languagetool-java, hash for LanguageTool-1.1.1.zip failed!
nswagstudio: couldn't match 'NSwag v([\d.]+)\s*\(Build\s*(\d+)\)' in https://api.github.com/repos/RicoSuter/NSwag/releases/latest
mupdf: couldn't match 'mupdf-([\w.-]+)-windows\.zip' in https://www.mupdf.com/downloads/index.html
nexus-repository-oss: 3.37.3-02 (scoop version is 3.37.0-01) autoupdate available
Autoupdating nexus-repository-oss
Searching hash for nexus-1.1.1-win64.zip in https://download.sonatype.com/nexus/1/nexus-1.1.1-win64.zip.sha1
The remote server returned an error: (404) Not Found.
URL https://download.sonatype.com/nexus/1/nexus-1.1.1-win64.zip.sha1 is not valid
Could not find hash in https://download.sonatype.com/nexus/1/nexus-1.1.1-win64.zip.sha1
Downloading nexus-1.1.1-win64.zip to compute hashes!
The remote server returned an error: (404) Not Found.
```

**Comment by rashil2000** [comment]
That's pretty weird!

The only change done to checkver.ps1 in past few days is this:

image

But it's not causing any problems in Main or Versions...

**Pull request merged**